src/OrderDetails.js

```diff
@@ -21,6 +21,7 @@ const OrderDetails = ({ order }) => {
     <Card>
       <Card.Header>
         <Row className="align-items-center">
+          <span className="rounded-pill p-0 ps ms-1" style={{ height: "50px", width: "8px", backgroundColor: order.car.color }}></span>
```

Comment:
To dobrze wygląda tylko na xxl. Nie da się zrobić height z flex align self stretch a width ustawić na rem zamiast px?

Reply:
Jak damy width na rem to się robi za cienkie na mniejszym ekranie, a i tak robimy desktop only.

Reply:
Może być px, a ten stretch? Nawet jak się lekko zmniejszy ekran to zmienia się breakpoint i nie zajmuje całej wysokości.

Reply:
Chociaż myślę że rem powinien działać na wszystkich ekranach bo jest właśnie do tego zrobiony.

Reply:
Stretch działa, ale rem się tak zwęża z czasem że nic nie widać.

Reply:
Zauważyłem, że grid powoduje takie zmniejszenie długości.

![image](https://user-images.githubusercontent.com/17147479/233349561-32fe1e93-df6b-4878-96fa-b3f689780e2a.png)

Na dole też ustawiony rem..

Reply:
Ok to px jest git.

Reply:
Na górze i na dole jest rem. To kwestia tego że górny rem jest zgniatany przez grida. Gdyby się dostosowało mobilnie to się powinno naprawić.